### PR TITLE
Temporary exclude numpy 1.17.* take 2

### PR DESCRIPTION
**Pull request by ericpre**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -36,7 +36,7 @@ outputs:
         - ipython
         - jupyter
         - matplotlib >2.2.2
-        - numpy >1.13,!=1.17
+        - numpy >1.13,!=1.17.*
```
Comment (francisco-dlp) [inline recipe/meta.yaml]: Wouldn't ``<1.17`` be better since it would be future-proof?

Comment (ericpre) [reply recipe/meta.yaml]: Yes, I was testing this more or less out of curiosity!

**Comment by ericpre** [comment]
Instead of excluding more recent version of numpy, which I am not very keen on, shall we make a patch release of hyperspy?

**Comment by francisco-dlp** [comment]
I suggest to go for both: release this new build now and a patch release asap.

**Comment by francisco-dlp** [comment]
@ericpre, it seems that this is building now. Do you agree to merge it once all the builds pass?

**Comment by ericpre** [comment]
Yes, even if I dislike pinning like this (in particular numpy), but since 1.5.2 will be released soon, this is fine.
Anyway if someone insist to install 1.5.1 (I can't think of any reason to want to do that!) without the pinning it is still possible to install one of the previous build and set one of numpy environment variable as described in https://github.com/hyperspy/hyperspy/pull/2247#issue-313534172.

The dask issue only affect loading velox emd file lazily (fairly minor) but it needs to be pinned to get the test suite to pass. We could neglect this issue and skip the test suite to avoid pinning dask too, now that we are sure that the tests pass.

**Comment by francisco-dlp** [comment]
I think that, even if what's broken with the latest dask is minor, it is still broken, so I am for constraining dask too. As you note, it is very unlikely that somebody will want to install 1.5.1 instead of 1.5.2 or newer after 1.5.2 is released, but I think that it is good practice to leave things tidy just in case, specially when it takes very little effort to achieve it such as in this case.

**Pull request merged**